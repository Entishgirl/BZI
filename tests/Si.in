&CONTROL
    pseudo_dir  = '/fslhome/jeremy12/quantum_espresso/integration_convergence/Si'
    calculation = 'scf',
    prefix = 'Si',
 /
 
&SYSTEM
    ibrav =  2,
    celldm(1) = 10.26,
    nat =  2,
    ntyp = 1,
    ecutwfc = 20
    occupations = 'tetrahedra'
    lspinorb = .true.
    noncolin = .true.
 /
 
&ELECTRONS
    mixing_beta = 0.7
    conv_thr = 1.0d-8
 /

ATOMIC_SPECIES
 Si 28.086  Si.rel-pbe-rrkj.UPF

ATOMIC_POSITIONS (alat)
 Si 0.0 0.0 0.0
 Si 0.25 0.25 0.25

K_POINTS (automatic)
 6 6 6 1 1 1
