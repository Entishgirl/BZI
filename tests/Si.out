
     Program PWSCF v.6.1 (svn rev. 13369) starts on  3Oct2017 at 12:21:33 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         295     295    109                 3287     3287     701



     bravais-lattice index     =            2
     lattice parameter (alat)  =      10.2600  a.u.
     unit-cell volume          =     270.0114 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=           16
     kinetic-energy cutoff     =      20.0000  Ry
     charge density cutoff     =      80.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Non magnetic calculation with spin-orbit


     celldm(1)=  10.260000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Si read from file:
     /fslhome/jeremy12/quantum_espresso/integration_convergence/Si/Si.rel-pbe-rrkj.UPF
     MD5 check sum: 90aaa04edacc1752306888b309598d47
     Pseudo is Ultrasoft + core correction, Zval =  4.0
     Generated using "atomic" code by A. Dal Corso  v.5.0.2 svn rev. 9415
     Using radial grid of 1141 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   1
                l(6) =   1
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Si             4.00    28.08600     Si( 1.00)

     48 Sym. Ops., with inversion, found (24 have fractional translation)



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Si  tau(   2) = (   0.2500000   0.2500000   0.2500000  )

     number of k points=    28 (tetrahedron method)
                       cart. coord. in units 2pi/alat
        k(    1) = (  -0.0833333   0.0833333   0.0833333), wk =   0.0092593
        k(    2) = (  -0.2500000   0.2500000  -0.0833333), wk =   0.0277778
        k(    3) = (  -0.4166667   0.4166667  -0.2500000), wk =   0.0277778
        k(    4) = (   0.4166667  -0.4166667   0.5833333), wk =   0.0277778
        k(    5) = (   0.2500000  -0.2500000   0.4166667), wk =   0.0277778
        k(    6) = (   0.0833333  -0.0833333   0.2500000), wk =   0.0277778
        k(    7) = (  -0.0833333   0.4166667   0.0833333), wk =   0.0277778
        k(    8) = (  -0.2500000   0.5833333  -0.0833333), wk =   0.0555556
        k(    9) = (   0.5833333  -0.2500000   0.7500000), wk =   0.0555556
        k(   10) = (   0.4166667  -0.0833333   0.5833333), wk =   0.0555556
        k(   11) = (   0.2500000   0.0833333   0.4166667), wk =   0.0555556
        k(   12) = (  -0.0833333   0.7500000   0.0833333), wk =   0.0277778
        k(   13) = (   0.7500000  -0.0833333   0.9166667), wk =   0.0555556
        k(   14) = (   0.5833333   0.0833333   0.7500000), wk =   0.0555556
        k(   15) = (   0.4166667   0.2500000   0.5833333), wk =   0.0555556
        k(   16) = (  -0.0833333  -0.9166667   0.0833333), wk =   0.0277778
        k(   17) = (  -0.2500000  -0.7500000  -0.0833333), wk =   0.0555556
        k(   18) = (  -0.0833333  -0.5833333   0.0833333), wk =   0.0277778
        k(   19) = (  -0.2500000   0.2500000   0.2500000), wk =   0.0092593
        k(   20) = (  -0.4166667   0.4166667   0.0833333), wk =   0.0277778
        k(   21) = (   0.4166667  -0.4166667   0.9166667), wk =   0.0277778
        k(   22) = (   0.2500000  -0.2500000   0.7500000), wk =   0.0277778
        k(   23) = (  -0.2500000   0.5833333   0.2500000), wk =   0.0277778
        k(   24) = (   0.5833333  -0.2500000   1.0833333), wk =   0.0555556
        k(   25) = (   0.4166667  -0.0833333   0.9166667), wk =   0.0555556
        k(   26) = (  -0.2500000  -1.0833333   0.2500000), wk =   0.0277778
        k(   27) = (  -0.4166667   0.4166667   0.4166667), wk =   0.0092593
        k(   28) = (   0.4166667  -0.4166667   1.2500000), wk =   0.0277778

     Dense  grid:     3287 G-vectors     FFT dimensions: (  24,  24,  24)

     Estimated max dynamical RAM per process >      14.54MB

     Check: negative/imaginary core charge=   -0.000010    0.000000

     Initial potential from superposition of free atoms

     starting charge    7.99890, renormalised to    8.00000
     Starting wfc are   16 randomized atomic wfcs

     total cpu time spent up to now is        1.8 secs

     Self-consistent Calculation

     iteration #  1     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.6

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  7.16E-04,  avg # of iterations =  1.1

     total cpu time spent up to now is        4.9 secs

     total energy              =     -22.83309573 Ry
     Harris-Foulkes estimate   =     -22.84662444 Ry
     estimated scf accuracy    <       0.05679866 Ry

     iteration #  2     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  7.10E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        5.9 secs

     total energy              =     -22.83547795 Ry
     Harris-Foulkes estimate   =     -22.83554501 Ry
     estimated scf accuracy    <       0.00211589 Ry

     iteration #  3     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.64E-05,  avg # of iterations =  3.0

     total cpu time spent up to now is        6.9 secs

     total energy              =     -22.83556329 Ry
     Harris-Foulkes estimate   =     -22.83555863 Ry
     estimated scf accuracy    <       0.00005124 Ry

     iteration #  4     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  6.40E-07,  avg # of iterations =  2.8

     total cpu time spent up to now is        8.1 secs

     total energy              =     -22.83557194 Ry
     Harris-Foulkes estimate   =     -22.83557185 Ry
     estimated scf accuracy    <       0.00000011 Ry

     iteration #  5     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.41E-09,  avg # of iterations =  3.6

     total cpu time spent up to now is        9.4 secs

     total energy              =     -22.83557222 Ry
     Harris-Foulkes estimate   =     -22.83557221 Ry
     estimated scf accuracy    <       0.00000001 Ry

     iteration #  6     ecut=    20.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.34E-10,  avg # of iterations =  2.0

     total cpu time spent up to now is       10.5 secs

     End of self-consistent calculation

          k =-0.0833 0.0833 0.0833 (   405 PWs)   bands (ev):

    -5.6605  -5.6605   5.3661   5.3661   6.0681   6.0681   6.0995   6.0995
     8.8350   8.8350   8.9953   8.9953   9.0182   9.0182  10.2406  10.2406

          k =-0.2500 0.2500-0.0833 (   399 PWs)   bands (ev):

    -5.1815  -5.1815   3.2771   3.2771   4.8357   4.8357   5.8719   5.8719
     8.9453   8.9453   9.4044   9.4044   9.7677   9.7677  11.9663  11.9663

          k =-0.4167 0.4167-0.2500 (   409 PWs)   bands (ev):

    -4.0830  -4.0830   0.4992   0.4992   4.3700   4.3700   5.2281   5.2281
     8.4364   8.4364   9.7904   9.7904   9.8333   9.8333  13.7227  13.7227

          k = 0.4167-0.4167 0.5833 (   407 PWs)   bands (ev):

    -3.3787  -3.3787  -0.6530  -0.6530   4.4378   4.4378   4.8442   4.8442
     8.2785   8.2785   9.5055   9.5055   9.9355   9.9355  14.1539  14.1539

          k = 0.2500-0.2500 0.4167 (   405 PWs)   bands (ev):

    -4.4974  -4.4974   1.3507   1.3507   4.8992   4.8992   4.9459   4.9459
     8.5224   8.5224   9.5331   9.5331  10.3314  10.3314  12.2212  12.2212

          k = 0.0833-0.0833 0.2500 (   401 PWs)   bands (ev):

    -5.4187  -5.4187   4.2945   4.2945   5.4292   5.4292   5.6225   5.6225
     8.5420   8.5420   9.4551   9.4551   9.9176   9.9176  11.1241  11.1241

          k =-0.0833 0.4167 0.0833 (   404 PWs)   bands (ev):

    -4.9392  -4.9392   3.0947   3.0947   4.6763   4.6763   4.6974   4.6974
     7.9550   7.9550  10.1309  10.1309  10.9287  10.9287  11.1519  11.1519

          k =-0.2500 0.5833-0.0833 (   409 PWs)   bands (ev):

    -4.0241  -4.0241   1.2470   1.2470   3.3506   3.3506   4.3035   4.3035
     8.3569   8.3569   9.9889   9.9889  10.8014  10.8014  12.2697  12.2697

          k = 0.5833-0.2500 0.7500 (   409 PWs)   bands (ev):

    -2.8995  -2.8995  -0.5176  -0.5176   2.7481   2.7481   4.0738   4.0738
     8.6070   8.6070  10.1768  10.1768  12.1452  12.1452  12.5239  12.5239

          k = 0.4167-0.0833 0.5833 (   411 PWs)   bands (ev):

    -3.6270  -3.6270   0.5489   0.5489   2.7031   2.7031   4.5651   4.5651
     9.2154   9.2154   9.8443   9.8443  11.5329  11.5329  12.0166  12.0166

          k = 0.2500 0.0833 0.4167 (   404 PWs)   bands (ev):

    -4.7125  -4.7125   2.2501   2.2501   4.0530   4.0530   5.2409   5.2409
     8.8521   8.8521   9.7457   9.7457  10.8044  10.8044  11.1450  11.1450

          k =-0.0833 0.7500 0.0833 (   413 PWs)   bands (ev):

    -3.3093  -3.3093   0.4207   0.4207   3.3128   3.3128   3.6562   3.6562
     7.1201   7.1201   8.0991   8.0991  13.8354  13.8354  13.8858  13.8858

          k = 0.7500-0.0833 0.9167 (   411 PWs)   bands (ev):

    -2.1063  -2.1063  -0.9255  -0.9255   2.4633   2.4633   3.2456   3.2456
     7.7481   7.7481   8.8794   8.8794  13.3672  13.3672  14.8470  14.8470

          k = 0.5833 0.0833 0.7500 (   405 PWs)   bands (ev):

    -2.4774  -2.4774  -0.5920  -0.5920   1.9779   1.9779   3.7956   3.7956
     8.3730   8.3730  11.3050  11.3050  12.2999  12.2999  12.9768  12.9768

          k = 0.4167 0.2500 0.5833 (   407 PWs)   bands (ev):

    -3.5134  -3.5134  -0.1459  -0.1459   3.6560   3.6560   4.6641   4.6641
     8.8363   8.8363   9.4405   9.4405  10.7834  10.7834  13.0097  13.0097

          k =-0.0833-0.9167 0.0833 (   413 PWs)   bands (ev):

    -2.1995  -2.1995  -0.9339  -0.9339   3.0411   3.0411   3.4394   3.4394
     7.0053   7.0053   7.5772   7.5772  15.3237  15.3237  15.4212  15.4212

          k =-0.2500-0.7500-0.0833 (   410 PWs)   bands (ev):

    -3.1393  -3.1393   0.1953   0.1953   2.7890   2.7890   3.5883   3.5883
     7.9606   7.9606   9.2491   9.2491  12.0055  12.0055  13.8328  13.8328

          k =-0.0833-0.5833 0.0833 (   413 PWs)   bands (ev):

    -4.2310  -4.2310   1.7917   1.7917   3.8625   3.8625   4.0815   4.0815
     7.4443   7.4443   9.0175   9.0175  12.2888  12.2888  12.4863  12.4863

          k =-0.2500 0.2500 0.2500 (   401 PWs)   bands (ev):

    -4.9508  -4.9508   2.2912   2.2912   5.4474   5.4474   5.4788   5.4788
     8.2541   8.2541   9.7311   9.7311   9.7529   9.7529  13.1357  13.1357

          k =-0.4167 0.4167 0.0833 (   404 PWs)   bands (ev):

    -4.2680  -4.2680   1.3580   1.3580   3.3148   3.3148   5.2931   5.2931
     9.0155   9.0155  10.2401  10.2401  10.4188  10.4188  12.3661  12.3661

          k = 0.4167-0.4167 0.9167 (   406 PWs)   bands (ev):

    -3.0816  -3.0816  -0.0873  -0.0873   2.1993   2.1993   4.4988   4.4988
     8.2576   8.2576  11.5661  11.5661  11.6851  11.6851  13.4208  13.4208

          k = 0.2500-0.2500 0.7500 (   412 PWs)   bands (ev):

    -3.0296  -3.0296  -0.1669  -0.1669   2.6825   2.6825   4.0125   4.0125
     7.7539   7.7539  10.6924  10.6924  11.9806  11.9806  12.1692  12.1692

          k =-0.2500 0.5833 0.2500 (   412 PWs)   bands (ev):

    -3.8415  -3.8415   0.5447   0.5447   3.7865   3.7865   4.3888   4.3888
     8.2437   8.2437  10.4944  10.4944  11.1416  11.1416  11.2952  11.2952

          k = 0.5833-0.2500 1.0833 (   409 PWs)   bands (ev):

    -2.8337  -2.8337  -0.1977  -0.1977   2.2993   2.2993   3.5858   3.5858
     9.4663   9.4663  10.0073  10.0073  10.9959  10.9959  13.7491  13.7491

          k = 0.4167-0.0833 0.9167 (   405 PWs)   bands (ev):

    -1.9976  -1.9976  -0.9333  -0.9333   2.1313   2.1313   2.9126   2.9126
     9.2635   9.2635  10.5329  10.5329  11.3481  11.3481  12.7404  12.7404

          k =-0.2500-1.0833 0.2500 (   410 PWs)   bands (ev):

    -2.2092  -2.2092  -0.8409  -0.8409   2.0042   2.0042   3.8166   3.8166
     7.4736   7.4736  10.4065  10.4065  13.3480  13.3480  13.5566  13.5566

          k =-0.4167 0.4167 0.4167 (   413 PWs)   bands (ev):

    -3.7511  -3.7511  -0.2501  -0.2501   5.0551   5.0551   5.0868   5.0868
     7.8210   7.8210   9.6283   9.6283   9.6439   9.6439  14.0372  14.0372

          k = 0.4167-0.4167 1.2500 (   406 PWs)   bands (ev):

    -3.1421  -3.1421  -0.5514  -0.5514   3.2271   3.2271   4.6271   4.6271
     8.4203   8.4203  10.3070  10.3070  10.6416  10.6416  14.2247  14.2247

     the Fermi energy is     6.1979 ev

!    total energy              =     -22.83557223 Ry
     Harris-Foulkes estimate   =     -22.83557223 Ry
     estimated scf accuracy    <          6.5E-10 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       5.12540110 Ry
     hartree contribution      =       1.09647549 Ry
     xc contribution           =     -12.25651922 Ry
     ewald contribution        =     -16.80092959 Ry

     convergence has been achieved in   6 iterations

     Writing output data file Si.save

     init_run     :      1.13s CPU      1.27s WALL (       1 calls)
     electrons    :      7.40s CPU      8.70s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.68s CPU      0.79s WALL (       1 calls)
     potinit      :      0.02s CPU      0.02s WALL (       1 calls)

     Called by electrons:
     c_bands      :      6.14s CPU      7.23s WALL (       7 calls)
     sum_band     :      1.15s CPU      1.32s WALL (       7 calls)
     v_of_rho     :      0.08s CPU      0.09s WALL (       7 calls)
     newd         :      0.03s CPU      0.06s WALL (       7 calls)
     mix_rho      :      0.01s CPU      0.02s WALL (       7 calls)

     Called by c_bands:
     init_us_2    :      0.08s CPU      0.08s WALL (     420 calls)
     cegterg      :      5.89s CPU      6.97s WALL (     196 calls)

     Called by sum_band:
     sum_band:bec :      0.04s CPU      0.04s WALL (     196 calls)
     addusdens    :      0.03s CPU      0.05s WALL (       7 calls)

     Called by *egterg:
     h_psi        :      5.30s CPU      6.28s WALL (     705 calls)
     s_psi        :      0.13s CPU      0.14s WALL (     705 calls)
     g_psi        :      0.06s CPU      0.06s WALL (     481 calls)
     cdiaghg      :      0.38s CPU      0.39s WALL (     649 calls)

     Called by h_psi:
     h_psi:pot    :      5.28s CPU      6.24s WALL (     705 calls)
     h_psi:calbec :      0.12s CPU      0.13s WALL (     705 calls)
     vloc_psi     :      5.01s CPU      5.97s WALL (     705 calls)
     add_vuspsi   :      0.15s CPU      0.14s WALL (     705 calls)

     General routines
     calbec       :      0.16s CPU      0.16s WALL (     901 calls)
     fft          :      0.02s CPU      0.03s WALL (     135 calls)
     fftw         :      4.75s CPU      5.82s WALL (   39420 calls)
     davcio       :      0.00s CPU      0.01s WALL (      28 calls)

     Parallel routines
     fft_scatter  :      0.52s CPU      0.77s WALL (   39555 calls)

     PWSCF        :     9.77s CPU        11.40s WALL


   This run was terminated on:  12:21:44   3Oct2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
